### PR TITLE
(blog) Draft of June Gatsby Gazette

### PR DESCRIPTION
**Pull request by hashimwarren**
June's Gatsby Gazette covers exciting updates from Gatsby and the Gatsby ecosystem

### CODE REVIEW

File: docs/blog/gazette-june-2020/index.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+The efforts above are just the beginning of the work were doing to ensure Black lives matter in our world, our industry, and our company.
+
+## 🚀 New in Gatsby and Gatsby Cloud!
+
+### New Site Workflow in Gatsby Cloud
+
+You can now create multiple sites from the same repository in Gatsby Cloud. This new feature enables you to set up multiple configurations for the same repo (for example, to set up a long-lived staging branch).
```
Comment (ashumz) [inline docs/blog/gazette-june-2020/index.md]:
```suggestion
You can now create multiple sites from the same repository in Gatsby Cloud. This new feature enables you to set up multiple configurations for the same repo (for example, to set up a long-lived staging branch or to build multiple sites in the same repository independently.)
```

Comment (hashimwarren) [reply docs/blog/gazette-june-2020/index.md]: Updates with your suggestion

File: docs/blog/gazette-june-2020/index.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+---
+title: Gatsby Gazette for June 2020 - The Black Lives Matter Edition
+date: 2020-06-30
```
Comment (ashumz) [inline docs/blog/gazette-june-2020/index.md:3]: update date
```suggestion
date: 2020-07-01
```

File: docs/blog/gazette-june-2020/index.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+### 3 new CMS's (plus 1) in Gatsby Cloud
+
+We've made it easier to use [Gatsby Cloud](https://gatsbyjs.com) with three amazing CMS's:
+
+1. [Kentico Kontent](https://www.gatsbyjs.com/guides/kentico-kontent/)
+2. [Flotiq](https://www.gatsbyjs.com/guides/flotiq/)
+3. [AgilityCMS](https://www.gatsbyjs.com/guides/agility-cms/).
+
+Plus - Gatsby Cloud also has [an improved Drupal integration](https://youtu.be/Mm6wrDr2DBE) (Check it out [here](https://www.drupal.org/project/gatsby).
```
Comment (ashumz) [inline docs/blog/gazette-june-2020/index.md:32]:
```suggestion
Plus - Gatsby Cloud also has [an improved Drupal integration](https://youtu.be/Mm6wrDr2DBE) (Check it out [here](https://www.drupal.org/project/gatsby).)
```

**Pull request merged**